kotlin-source/src/main/kotlin/com/example/flow/ExampleFlow.kt

```diff
@@ -41,13 +43,15 @@ object ExampleFlow {
             object GENERATING_TRANSACTION : ProgressTracker.Step("Generating transaction based on new IOU.")
             object VERIFYING_TRANSACTION : ProgressTracker.Step("Verifying contract constraints.")
             object SIGNING_TRANSACTION : ProgressTracker.Step("Signing transaction with our private key.")
-            object SENDING_TRANSACTION : ProgressTracker.Step("Sending proposed transaction to recipient for review.")
+            object GATHERING_SIGS : ProgressTracker.Step("Gathering the counterparty's signature.")
```

Comment:
you can override the child progress tracker with the `CollectsignaturesFlow.tracker()`, likewise for the Finality flow. See the IOUv2 app for an example.